README.MD

```diff
@@ -35,7 +35,9 @@ The site was built with Ruby.
 
 ### Key Features <a name="key-features"></a>
 
-- N/A
+- Person Class.
+- Student Class.
+- Teacher Class.
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
```

Comment:
- [ ] To improve your documentation to be more professional and descriptive, I think it would be better if you update your `Getting Started` section by providing all needed information by users in order to run your app such as `tools needed`, `how to access the cloned dir`, `use or run your app`,... To help you, I recommend you refer **[this gist_file as template](https://gist.github.com/tresorsawasawa/0eba0570695aef068a7837c349f94737)**.